lib/origen_testers/smartest_based_tester/base/test_methods/base_tml.rb

```diff
@@ -12,6 +12,11 @@ def initialize(test_methods)
           end
 
           def method_missing(method, *args, &block)
+            tml_class_name = self.class.name.to_s.split('::')[-1]
+            # This enables passing camel case test methods for dc_tml and ac_tml
+            unless tml_class_name == 'CustomTml'
```

Comment:
Don't understand what this line is doing, why is a specific library name handled here?

Reply:
@ginty The logic prevents altering any custom TMLs.  My understanding is that a custom TML should be handled as-is.

Reply:
I see, sorry I missed the line before, I didn't realize this was a Ruby class reference.
Anyhow, I think we should treat custom TMLs the same.